packages/twilio-run/src/config/utils/credentials.ts

```diff
@@ -30,49 +33,36 @@ export async function getCredentialsFromFlags<
   externalCliOptions?: ExternalCliOptions
 ): Promise<Credentials> {
   // default Twilio CLI credentials (4) or empty string (5)
-  let accountSid =
-    (externalCliOptions &&
-      !(externalCliOptions.profile || externalCliOptions.project) &&
-      externalCliOptions.username) ||
-    '';
-  let authToken =
-    (externalCliOptions &&
-      !(externalCliOptions.profile || externalCliOptions.project) &&
-      externalCliOptions.password) ||
-    '';
+  let username = '';
+  let password = '';
 
-  // env file content (3)
-  if (envVariables.ACCOUNT_SID) {
-    debug('Override value with .env ACCOUNT_SID value');
-    accountSid = envVariables.ACCOUNT_SID;
-  }
-  if (envVariables.AUTH_TOKEN) {
-    debug('Override value with .env AUTH_TOKEN value');
-    authToken = envVariables.AUTH_TOKEN;
-  }
-
-  // specific profile specified. override both credentials (2)
-  if (
-    externalCliOptions &&
-    (externalCliOptions.profile || externalCliOptions.project)
-  ) {
-    debug('Values read from explicit CLI profile');
-    accountSid = externalCliOptions.username;
-    authToken = externalCliOptions.password;
+  if (externalCliOptions) {
+    debug('Using account credentials from Twilio CLI');
+    username = externalCliOptions.username;
+    password = externalCliOptions.password;
+  } else {
+    if (envVariables.ACCOUNT_SID) {
+      debug('Using Account SID from env variables');
+      username = envVariables.ACCOUNT_SID;
+    }
+    if (envVariables.AUTH_TOKEN) {
+      debug('Using Auth Token from env variables');
+      password = envVariables.AUTH_TOKEN;
+    }
   }
 
   // specific flag passed. override for that flag (1)
   if (flags.accountSid) {
     debug('Override accountSid with value from flag');
-    accountSid = flags.accountSid;
+    username = flags.accountSid;
```

Comment:
Should we rename for everything but `init` the flags to `username` and `password` as opposed to `accountSid`and `authToken` to create consistency here.

The main point would be because with the new config changes people could specify those credentials in a `.twilioserverlessrc` file if they wanted to.

Reply:
Flags are all updated to username and password in twilio-run now. `create-twilio-function` shall indeed continue to expect account sid and password.